src/test/server/Server_test.cpp

```diff
@@ -367,10 +370,98 @@ class Server_test : public beast::unit_test::suite
     }
 
     void
+    testBadConfig ()
```

Comment:
I think that capturing the output would also allow you to check that exceptions are being thrown were you expect.